### PR TITLE
Katello 3.7 RC1 Release Notes

### PR DESCRIPTION
**Pull request by jturel**

### CODE REVIEW

File: plugins/katello/3.7/release_notes/release_notes.md [hunk 1]
```diff
@@ -4,71 +4,36 @@ title: Release Notes
...
 
-# Katello 3.6 (Imperial IPA) Release Notes
+# Katello 3.7 (Blonde Ale) Release Notes
 
-For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-3.6/CHANGELOG.md).
+For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-3.7/CHANGELOG.md).
 
-## Warning
+## Known RC1 Issues
+* Upgrades from 3.6 to 3.7 are not fully functional due to a failing database migration
```
Comment (bbuckingham) [inline plugins/katello/3.7/release_notes/release_notes.md]: Should we reference redmine issues for the known issues that are being worked?

File: plugins/katello/3.7/release_notes/release_notes.md [hunk 2]
```diff
@@ -4,71 +4,36 @@ title: Release Notes
...
+### New Red Hat Repositories Page
+Completely redesigned with an eye toward usability and functionality.
 
-## Deprecation Warnings
+### New Upstream Subscriptions API
+For users with Red Hat content we now provide an API for managing subscriptions against existing manifests. Check the API documentation for details.
 
-#### File Repository Paths on Upgrade
+### Content Views
+Composite content views will now be auto-published when one of its components is changed.
```
Comment (bbuckingham) [inline plugins/katello/3.7/release_notes/release_notes.md]: May want to change 'will' to 'can'.  It is an optional feature that a user may take advantage of.

File: plugins/katello/3.7/release_notes/release_notes.md [hunk 3]
```diff
@@ -4,71 +4,36 @@ title: Release Notes
...
+### Client Tooling
+Zypper and DNF plugins have been added to support errata applicability similar to Yum clients
 
-Old file repositories can be cleaned up at a user's discretion to reclaim disk space by running:
+### Backup and Restore Utilities
+We are beginning to adopt [Foreman Maintain](https://theforeman.org/plugins/foreman_maintain/0.2/index.html) for Katello. Our first endeavor has been to utilize it for backup and restore capabilities.
```
Comment (bbuckingham) [inline plugins/katello/3.7/release_notes/release_notes.md:31]: Has katello-service also been fully moved over to foreman-maintain?  (e.g. foreman-maintain service vs katello-service)

File: plugins/katello/3.7/release_notes/release_notes.md [hunk 4]
```diff
@@ -4,71 +4,36 @@ title: Release Notes
...
 
-#### File Repository Paths on Upgrade
+### Content Views
+Composite content views will now be auto-published when one of its components is changed.
 
-Up until now file repositories were not able to be synced out to a proxy due to the URL path that the published file repositories were generated at. Katello 3.6 modifies the published path to mirror paths similar to how yum and puppet modules are published. This allows proper HTTPS serving of files repositories and syncing them out to proxy along with a more obvious and readable path. The downside of this needed change is that all file repositories are republished to a new path rendering all previous paths incorrect for any changes to the file repositories from the point of upgrade forward. The original file repository paths are left in tact, snapshotted at the point of the upgrade. This will allow a small window to update any infrastructure that relies on those file paths.
+### Client Tooling
+Zypper and DNF plugins have been added to support errata applicability similar to Yum clients
```
Comment (bbuckingham) [inline plugins/katello/3.7/release_notes/release_notes.md]: Are we also delivering subscription-manager for suse clients as part of the foreman/katello repos?

File: plugins/katello/3.7/release_notes/release_notes.md [hunk 5]
```diff
@@ -4,71 +4,36 @@ title: Release Notes
...
 
-Old file repositories can be cleaned up at a user's discretion to reclaim disk space by running:
+### Backup and Restore Utilities
+We are beginning to adopt [Foreman Maintain](https://theforeman.org/plugins/foreman_maintain/0.2/index.html) for Katello. Our first endeavor has been to utilize it for backup and restore capabilities.
 
-```bash
-foreman-rake katello:cleanup_old_file_repos
-```
-
-## Contributors
```
Comment (akofink) [inline plugins/katello/3.7/release_notes/release_notes.md]: What's the plan here? No more contributors section?

File: plugins/katello/3.7/release_notes/release_notes.md (hunk 3 above)
Comment (jturel) [reply plugins/katello/3.7/release_notes/release_notes.md:31]: I don't think so - can you comment on that @johnpmitsch ?

File: plugins/katello/3.7/release_notes/release_notes.md (hunk 1 above)
Comment (jturel) [reply plugins/katello/3.7/release_notes/release_notes.md]: Yes! Fortunately I think we can resolve the client tool repos prior to announcement which leaves one issue outstanding

File: plugins/katello/3.7/release_notes/release_notes.md (hunk 5 above)
Comment (jturel) [reply plugins/katello/3.7/release_notes/release_notes.md]: Generating it :)

File: plugins/katello/3.7/release_notes/release_notes.md (hunk 3 above)
Comment (johnpmitsch) [reply plugins/katello/3.7/release_notes/release_notes.md:31]: [Katello-service will redirect to foreman-maintain](https://github.com/theforeman/foreman-packaging/pull/2585) starting in 3.8. In 3.7, katello-service is the original script and lives alongside foreman-maintain service.

File: plugins/katello/3.7/release_notes/release_notes.md [hunk 6]
```diff
@@ -4,71 +4,77 @@ title: Release Notes
...
- - Shira Maximov
- - Stephen Benjamin
- - Tom McKay
- - Tomas Strachota
- - Tomer Brisker
- - Walden Raines
- - Zjhuntin
- - aagrawal
- - kgaikwad
+The old tools are completely removed so please be sure to update any cron jobs or scripts using them.
```
Comment (johnpmitsch) [inline plugins/katello/3.7/release_notes/release_notes.md]: Can you be specific about `old tools` and say `katello-backup and katello-restore`?

Comment (johnpmitsch) [reply plugins/katello/3.7/release_notes/release_notes.md]: Also could we put this in bold? Its a pretty important point so want to make sure users don't miss it

File: plugins/katello/3.7/release_notes/release_notes.md [hunk 7]
```diff
@@ -4,71 +4,77 @@ title: Release Notes
 version: 3.7
 ---
 
-# Katello 3.6 (Imperial IPA) Release Notes
+# Katello 3.7 (Blonde Ale) Release Notes
```
Comment (johnpmitsch) [inline plugins/katello/3.7/release_notes/release_notes.md:7]: would prefer `Mike's hard lemonade` :stuck_out_tongue_winking_eye:

Comment (jturel) [reply plugins/katello/3.7/release_notes/release_notes.md:7]: Lol

File: plugins/katello/3.7/release_notes/release_notes.md [hunk 8]
```diff
@@ -4,71 +4,77 @@ title: Release Notes
...
 
-### Content View Optimizations
 
-Performance improvements across the board have brought up to a 50% improvement in Content View Publishes, 95% improvement in Promotions, and a 95% improvement in Composite Content View publishes.
+### New Red Hat Repositories Page
+Completely redesigned with an eye toward usability and functionality.
 
-## Deprecation Warnings
+### New Upstream Subscriptions API
+For users with Red Hat content we now provide an API for managing subscriptions against existing manifests. Check the API documentation for details.
```
Comment (johnpmitsch) [inline plugins/katello/3.7/release_notes/release_notes.md:22]: Is the page available in the lab features in rc1? If so, should we mention that as a "tech preview" kind of thing?

Comment (jturel) [reply plugins/katello/3.7/release_notes/release_notes.md:22]: That was my plan but it is broken in RC1, so holding off in my opinion

**Review by akofink — APPROVED** [review]
My concerns have been addressed. ACK from me!

File: plugins/katello/3.7/release_notes/release_notes.md (hunk 4 above)
Comment (jturel) [reply plugins/katello/3.7/release_notes/release_notes.md]: I think we rely on base repos or Red Hat repos for that as I don't see subscription-manager in our client repos. @ehelms can you validate that theory?

**Comment by jturel** [comment]
I think I've addressed all the feedback to this point (thanks!) - please take a look

**Review by johnpmitsch — APPROVED** [review]
Looks good!

**Comment by bbuckingham** [comment]
Ack from me as well

**Pull request merged**